boards/samr21-xpro/Makefile.dep

```diff
@@ -1,4 +1,4 @@
 ifneq (,$(filter gnrc_netif_default,$(USEMODULE)))
   USEMODULE += at86rf233
-  USEMODULE += gnrc_nomac
+  USEMODULE += gnrc_lwmac
```

Comment:
IMO we should discuss on the mailing list or in the next meeting what we want to configure as the default MAC.